pages/settings/interface/index.vue

```diff
@@ -1,9 +1,36 @@
 <script setup lang="ts">
+import type { mastodon } from 'masto'
 const { t } = useI18n()
 
 useHeadFixed({
   title: () => `${t('settings.interface.label')} | ${t('nav.settings')}`,
 })
+
+const loremStatus = computed(() => {
+  if (currentUser.value) {
+    const loremStatus: mastodon.v1.Status = {
+      account: currentUser.value!.account,
```

Comment:
maybe we should also have a fake user when there's no user signed in.

Reply:
Agreed that would be better, should we use the Elk account details as the fake user?

Reply:
Yes!